src/contexts/KeyboardContext.ts

```diff
@@ -76,6 +82,7 @@ export type KeyboardProps = {
   enableRecentlyUsed?: boolean
   categoryPosition?: CategoryPosition
   enableSearchBar?: boolean
+  customButton?: React.ReactNode[]
```

Comment:
and one more totally minor thing - with this one can add one or more buttons? What about making the prop name plural - `customButtons`? wdyt? 🤔

Reply:
Yes, that might make it clearer for developers. Changed